.github/workflows/pr_push.yml

```diff
@@ -1,93 +1,96 @@
-    cancel-in-progress: true
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
 
 permissions:
   contents: read
 
 jobs:
-    FastBuild:
-        name: Fast build
-        strategy:
-            matrix:
-                os: [ubuntu-latest]
-                nproc: ['$(nproc)']
-                pool_scalable: ['ON']
-                disjoint: ['ON', 'OFF']
-                jemalloc: ['ON']
-                include:
-                    - os: windows-latest
-                      nproc: $Env:NUMBER_OF_PROCESSORS
-                      pool_scalable: 'OFF'
-                      disjoint: 'OFF'
-                      jemalloc: 'OFF'
-        runs-on: ${{matrix.os}}
+  FastBuild:
+    name: Fast build
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        nproc: ['$(nproc)']
+        pool_scalable: ['ON']
+        disjoint: ['ON', 'OFF']
+        jemalloc: ['ON']
+        extra_build_options: ['-DCMAKE_BUILD_TYPE=Release -DUMF_BUILD_BENCHMARKS=ON -DUMF_BUILD_BENCHMARKS_MT=ON']
+        include:
+          - os: windows-latest
+            nproc: $Env:NUMBER_OF_PROCESSORS
+            pool_scalable: 'OFF'
+            disjoint: 'OFF'
+            jemalloc: 'OFF'
+    runs-on: ${{matrix.os}}
 
-        steps:
-        - name: Checkout repository
-          uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
 
-        - name: Install apt packages
-          if: matrix.os == 'ubuntu-latest'
-          run: |
-            sudo apt-get update
-            sudo apt-get install -y cmake libjemalloc-dev libhwloc-dev libnuma-dev libtbb-dev
+    - name: Install apt packages
+      if: matrix.os == 'ubuntu-latest'
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y cmake libjemalloc-dev libhwloc-dev libnuma-dev libtbb-dev
 
-        - name: Configure CMake
-          run: >
-            cmake
-            -B ${{github.workspace}}/build
-            -DCMAKE_BUILD_TYPE=Release
-            -DUMF_FORMAT_CODE_STYLE=OFF
-            -DUMF_DEVELOPER_MODE=ON
-            -DUMF_ENABLE_POOL_TRACKING=ON
-            -DUMF_BUILD_LIBUMF_POOL_SCALABLE=${{matrix.pool_scalable}}
-            -DUMF_BUILD_LIBUMF_POOL_DISJOINT=${{matrix.disjoint}}
-            -DUMF_BUILD_LIBUMF_POOL_JEMALLOC=${{matrix.jemalloc}}
-            -DUMF_BUILD_TESTS=OFF
-            -DUMF_BUILD_EXAMPLES=ON
+    - name: Configure CMake
+      run: >
+        cmake
+        -B ${{github.workspace}}/build
+        -DUMF_FORMAT_CODE_STYLE=OFF
+        -DUMF_DEVELOPER_MODE=ON
+        -DUMF_ENABLE_POOL_TRACKING=ON
+        -DUMF_BUILD_LIBUMF_POOL_SCALABLE=${{matrix.pool_scalable}}
+        -DUMF_BUILD_LIBUMF_POOL_DISJOINT=${{matrix.disjoint}}
+        -DUMF_BUILD_LIBUMF_POOL_JEMALLOC=${{matrix.jemalloc}}
+        -DUMF_BUILD_TESTS=ON
```

Comment:
Is there any build for verifying the pure C build after this change? Don't we want to have one?